src/main/java/com/aws/greengrass/clientdevices/auth/iot/events/ThingEvent.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.clientdevices.auth.iot.events;
+
+import com.aws.greengrass.clientdevices.auth.api.DomainEvent;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.List;
+
+@AllArgsConstructor
+public class ThingEvent implements DomainEvent {
+    @Getter
+    private ThingEventType eventType;
+    @Getter
+    private String thingName;
+    @Getter
+    private List<String> attachedCertificateIds;
+
+    public enum ThingEventType {
+        THING_UPDATED
```

Comment:
I started with ADDED/UPDATED/DELETED, but we don't really have use cases for those other event types, so I removed them. This simplifies registry logic somewhat.

The idea is to try to model IoT Core registry events:
https://docs.aws.amazon.com/iot/latest/developerguide/registry-events.html

Reply:
No JSON needed at this point. We can remove the type enum